scripts/devcloud-test.sh

```diff
@@ -0,0 +1,44 @@
+#! /bin/bash
+# Copyright (C) 2023 Intel Corporation
+
+# Part of the Unified-Runtime Project, under the Apache License v2.0 with LLVM Exceptions.
+# See LICENSE.TXT
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+workspace=$1
+compiler_c=$2
+compiler_cxx=$3
+build_type=$4
+
+set -e
+hostname
+
+# SLURM/MPI integration is broken
+unset SLURM_TASKS_PER_NODE
+unset SLURM_JOBID
+
+source /opt/intel/oneapi/setvars.sh
+source ${workspace}/dpcpp_compiler/startup.sh
+export PATH=${workspace}/dpcpp_compiler/bin:$PATH
+export CPATH=${workspace}/dpcpp_compiler/include:$CPATH
+export LIBRARY_PATH=${workspace}/dpcpp_compiler/lib:$LIBRARY_PATH
+export LD_LIBRARY_PATH=${workspace}/dpcpp_compiler/lib:$LD_LIBRARY_PATH
```

Comment:
All these `export` steps are redundant as you already do this here:
```
source ${workspace}/dpcpp_compiler/startup.sh
```
See the contents of this script.

Reply:
I have changed it. There is no startup.sh script in the new dpc++ package.